docs/tutorials/ui-customization.md

```diff
@@ -17,7 +17,7 @@ const config = {
 ui.configure(config);
 ```
 
-Controls will fire a {@link shaka.ui.Controls.UIUpdatedEvent} event once the
+Controls will fire a [shaka.ui.Controls.UIUpdatedEvent][] event once the
```

Comment:
Instead of adding a link manually, we'll need to find out why the @link not working for UIUpdatedEvent.

Reply:
I couldn't find where its link was stored? Can you please tell me where links are stored for @link ?

Reply:
@michellezhuogg I have thought of some changes but where can I run this documentation?

Reply:
Hello, please confirm the link works on your local page, and we're good to go. Thank you!
You might be able to check your local page via:   
http://localhost/shaka/docs/api/tutorial-ui-customization.html

Reply:
The `@link` syntax is part of jsdoc.  When you run `./build/docs.py`, it will build the markdown docs into HTML in `docs/api/`.

Reply:
@joeyparrish @michellezhuogg Thank you very much for guiding after testing I found out this doesn't work we have to set the link manually because in the link after HTML there event which due to which it can't be executed using jsdoc syntax `@link` 'https://shaka-player-demo.appspot.com/docs/api/shaka.ui.Controls.html#.event:UIUpdatedEvent' . In addition, in the page https://raw.githubusercontent.com/google/shaka-player/master/docs/tutorials/ad_monetization.md links have been set manually `[shaka.ads.AdManager.ImaAdManagerLoadedEvent]: https://nightly-dot-shaka-player-demo.appspot.com/docs/api/shaka.ads.AdManager.html#.event:AdManagerLoadedEvent
[shaka.ads.AdManager.ImaStreamManagerLoadedEvent]: https://nightly-dot-shaka-player-demo.appspot.com/docs/api/shaka.ads.AdManager.html#.event:ImaStreamManagerLoadedEvent`. So, should I add the link manually the way I did earlier?

Reply:
We fixed the link via this [commit](https://github.com/google/shaka-player/commit/99314b733f74922e525b79be8af6bc70b6841ae0). Thank you for pointing out the not-working link for us!

Reply:
Welcome and Thanks, now I learned how it can be solved earlier I couldn't find it on the internet.